src/index.d.ts

```diff
@@ -114,6 +123,18 @@ declare module 'react-native-screens' {
      * @description When set to true, the status bar for this screen is hidden. Requires enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file. Defaults to `false`.
      */
     statusBarHidden?: boolean;
+    /**
+     * @description Controls in which orientation should the screen appear.
+     * @type "default" - resolves to "all" without "portrait_down"
+     * @type "all" – all orientations are permitted
+     * @type "portrait" – portrait orientations are permitted
+     * @type "portrait_up" – right-side portrait orientation is permitted
+     * @type "portrait_down" – upside-down portrait orientation is permitted
+     * @type "landscape" – landscape orientations are permitted
+     * @type "landscape_left" – landscape-left orientation is permitted
+     * @type "landscape_right" – landscape-right orientation is permitted
+     */
+    stackOrientationMask?: StackOrientationMaskTypes;
```

Comment:
What was the intention behind this naming? For me it sound inaccurate for two reasons:
1) why does it mention stack? I think it'd be more intuitive if it was a screen property that is relevant to a single screen and not to the whole stack.
2) Why does it say "mask"? Masks are typically used when you are allowed to select multiple options. Instead here we have a list of allowed options and only one can be selected (yes, some options allow for both right and left portrait but it does not matter in my opinion). In order to match the mask behavior we should only have base level options defined and allow them to be mixed (e.g. we have portrait_left and portrait_right and allow both to be used as an array or expressed like a bit mask)

Reply:
1. I tried to follow the props `stackAnimation` and `stackPresentation`, but I am aware that this name was maybe not the best in that case.
2. It is the leftover from using the "mask" options on the native side.

`screenOrientation` would be the best name probably?